.github/workflows/build-nym-vpn-app-windows-ev-sign.yml

```diff
@@ -237,10 +273,10 @@ jobs:
           TAURI_SIGNING_PRIVATE_KEY_PASSWORD: ${{ secrets.TAURI_KEY_PASSWORD }}
           APP_SENTRY_DSN: ${{ secrets.DESKTOP_JS_SENTRY_DSN }}
           RUSTFLAGS: "-L ${{ env.TAURI_SRC }}/x86_64-pc-windows-msvc -L ${{ env.TAURI_SRC }} -Clink-args=/LIBPATH:${{ env.TAURI_SRC }}/x64-${{ env.CPP_BUILD_MODES }}"
-          WINDOWS_CERTIFICATE: ${{ secrets.WINDOWS_SIGNING_PFX_BASE64 }}
-          WINDOWS_CERTIFICATE_PASSWORD: ${{ secrets.WINDOWS_SIGNING_PFX_PASSWORD }}
           NETWORK_ENV_SELECT: ${{ inputs.dev_mode == true }}
+          CODE_SIGN_TOOL_PATH: "C:\\actions-runner\\_work\\nym-vpn-client\\nym-vpn-client\\nym-vpn-app\\src-tauri\\"
```

Comment:
Same here, maybe use `${{ github.workspace }}` to obtain the prefix?

Reply:
No we can't.

`${{ github.workspace }}` give a path in the form of `C:\action-runner\_work\nym-vpn-client` since we're in windows.

Since we're running in bash, \n get interpreted as a line return.

Reply:
Errm I see.